### DISCUSSION TITLE
input 和 select 等表单场景增加内嵌 label 类型

### DISCUSSION THREAD

**Discussion by roy-kang** [DevCloudFE/vue-devui#1358] [Ideas]
如下图：
![image](https://user-images.githubusercontent.com/112255758/191935077-bff010ab-5106-4122-853b-ff37dc73bb68.png)

[labeledInput](https://demo.grapecity.com/wijmo/demos/Input/LabeledInput/vue)

输入框和选择框的 placeholder 就以 label 数据展示，在 active 和 focus 的状态下，label 就动态移动到当前组件的上方

这种形式在数据量非常多的时候是非常节省页面空间的，可以考虑下在 input、date 和 select 等输入型的组件上加上这个类型。

**Comment by kagol** [comment]
这个组件的体验很赞！我觉得可以支持！

**Comment by kagol** [comment]
已创建 issue #1365
